src/conf_mode/high-availability.py

```diff
@@ -171,6 +206,18 @@ def apply(ha):
         call(f'systemctl stop {service_name}')
         return None
 
+    # Check if IPv6 address is tentative T5533
+    for group, group_config in ha['vrrp']['group'].items():
+        if 'hello_source_address' in group_config:
+            if is_ipv6(group_config['hello_source_address']):
+                ipv6_address = group_config['hello_source_address']
+                interface = group_config['interface']
+                checks = 20
+                interval = 0.1
+                for _ in range(checks):
+                    if _is_ipv6_tentative(interface, ipv6_address):
+                        time.sleep(interval)
```

Comment:
This will block until the IPv6 address is no longer tentiative. Is this what we want?

Reply:
It should work like this
If `_is_ipv6_tentative` is `True` then we sleep and wait until the address is available/fully configured (tentative = False)
Only then we can start keepalived.

Otherwise, we get `FAULT`:
```
Aug 31 13:49:22 r14 Keepalived_vrrp[4771]: bind unicast_src 2001:db8::3 failed 99 - Cannot assign requested address
```